imports/wasi_http/wasi_streams/streams.go

```diff
@@ -0,0 +1,84 @@
+package wasi_streams
```

Comment:
Similar comment on naming consistency between the module and Go package here. How about renaming to `streams` since that's the name of the host module. It's a bit generic but so is `types` and the namespacing under `wasi_http` still clarifies what this is about I think.

Reply:
renamed to `streams` some of this will clean up a little bit once wasi preview2 is implemented, since streams are part of that.